src/translation-assistant.js

```diff
@@ -441,18 +464,26 @@ function translateMath(math, lang) {
  */
 function normalizeTranslatedMath(math, lang) {
 
-    // NOTE(danielhollas): Maybe we could apply this regardless of locales
-    // to make the code more simple? Otherwise, the lists of locales here
-    // needs to be in sync with the list in translateMath function
     const mathNormalizations = [
          // Strip superfluous curly braces around \\,
          // which is used as thousand separator in some locales
          // i.e. 10{,}200 can be translated as 10{\\,}200, but the curly
          // braces are not really needed.
          // To understand why braces are needed around comma, see:
          // https://tex.stackexchange.com/questions/303110/avoid-space-after-thousands-separator-in-math-mode#303127
-         {langs: THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES,
+         {langs: MATH_RULES_LOCALES['THOUSAND_SEP_AS_THIN_SPACE'],
             regex: /([0-9])\{\\,\}([0-9])(?=[0-9]{2})/g, replace: '$1\\,$2'},
+
+         // Allow translators to use a full space (~ in LaTeX)
+         // (but TA will always suggest thin space \,)
+         // We cannot allow a literal space here, cause Tex would ignore it
+         {langs: MATH_RULES_LOCALES['THOUSAND_SEP_AS_THIN_SPACE'],
+            regex: /([0-9])\{?~\}?([0-9])(?=[0-9]{2})/g, replace: '$1\\,$2'},
+         //TODO: Strip extra braces around a dot as a thousand separator
```

Comment:
Is this going to revisited in #14?

I'm concerned leaving commented code with a question like that. As context will be lost over time. If this is an open question I would suggest deleting this block and creating a JIRA issue on the IC board with all relevant information.

Reply:
Nope, #14 should precede this one. 

Yeah, this is kind of an open question, I will open it up on i18n forum with the relevant teams. I can delete the code, although I'd like to leave a NOTE at least about it. Would that be okay. In any case, don't plan on leaving this undecided for long.

Reply:
Yep a NOTE makes sense, thank you.